README.md

```diff
@@ -301,6 +302,19 @@ In the animation, cyan points are searched nodes.
 
 Its heuristic is 2D Euclid distance.
 
+### D\* algorithm
+
+This is a 2D grid based the shortest path planning with D star algorithm.
+
+![figure at master · nirnayroy/intelligentrobotics](https://github.com/nirnayroy/intelligent-robotics/blob/main/dstar.gif)
```

Comment:
[comment] I will change the link to my gif repository.

Reply:
Should I create a pull request to your gifs repository?

Reply:
Thanks. You don't need to do that. I can do that.